dev/build-tool-support/src/main/scala/com/lightbend/lagom/dev/Reloader.scala

```diff
@@ -51,6 +51,14 @@ object Reloader {
 
   private def urls(cp: Seq[File]): Array[URL] = cp.map(_.toURI.toURL).toArray
 
+  case class DevServerBinding(protocol: String, address: String, port: Int)
```

Comment:
Why do we need `ServiceBindingInfo` and this one? Isn't `ServiceBindingInfo` part of dev mode console logs?

Reply:
Made some improvements and `DevServerBinding` is reused inside `ServiceBindingInfo` making things a bit more clear (IMHO)

Reply:
@marcospereira, the code that fills in `DevServerBinding` has no knowledge about the name of the project it is generating the bindings for.

Once it gets generated and returned, the caller creates `ServiceBindingInfo` containing the name of the service plus any `DevServerBinding`.

We could have passed the name down, but I believe this is better as it is now.